Astral run - some adjustments (#242)
qwrtln/Homm3BG-mission-book: draft-scenarios/clash/astral_run.tex

\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 2P × Far (II-III) Map Tiles
  \item P × Near (IV-V) Map Tiles
  \item 6 × Near (IV-V) Map Tiles with Obelisk
  \item 1 × Center (VI-VII) Map Tile
\end{itemize}

\subsection*{\MakeUppercase{Victory Conditions}}
The game ends when any of the following conditions are met:

\begin{itemize}

>   \item One player has defeated each other player's Main Heroes once. \textit{That player wins the game immediately.}

Reviewer: ```suggestion
  \item One player has defeated each of the other players' Main Heroes once. \textit{That player wins the game immediately.}
```

  \item One player has flagged all 6 Obelisks. \textit{That player wins the game immediately.}
  \item At the end of Round 15.
\end{itemize}

\subsection*{\MakeUppercase{Victory Points}}
If no player has achieved an immediate victory, the player with the most Victory Points (VP) wins.

\begin{itemize}
  \item 5 VPs for the first player to flag the Center VII Field.
  \item 3 VPs for each enemy Main Hero defeated.
  \item 1 VP for each flagged Obelisk.
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}
\textbf{\nth{4} Round:}
\begin{itemize}
  \item Remove all Black Cubes from the Map except from Learning Stones.
\end{itemize}
\textbf{\nth{6} Round:}
\begin{itemize}
  \item Player(s) whose Main Hero has the least \svg{experience}, Search (2) \svg{artifact}.
\end{itemize}
\textbf{\nth{8} Round:}
\begin{itemize}
  \item Repeat event of Round 4.
\end{itemize}
\textbf{\nth{10} Round:}
\begin{itemize}

>   \item Player(s) whose Main Hero has the least \svg{experience}, gain 2 \svg{resource}.

Reviewer: Why "gain"?

\end{itemize}
\textbf{\nth{12} Round:}
\begin{itemize}
  \item Repeat event of Round 4.
\end{itemize}

\subsection*{\MakeUppercase{Additional Rules}}
\begin{itemize}

>   \item Blocked Field on the Map Tile in the center does not count as blocked.

Reviewer: ```suggestion
  \item The Blocked Field on the Map Tile in the center is not considered blocked.
```

  \item \textbf{Sanctuary:} Search (2) your Deck. \textit{Visitable once per Faction}.

>   \item \textbf{Obelisk:} Guarded by Level VI Neutrals. Search (2) \svg{skill}, \svg{spellpower} or \svg{artifact}. \textit{Can be flagged by multiple players.}

Reviewer: ```suggestion
  \item \textbf{Obelisk:} Guarded by Level VI Neutral Units. Search (2) \svg{skill}, \svg{spellpower} or \svg{artifact}. \textit{Can be flagged by multiple players.}
```

  \item Each \textbf{blocked Field} adjacent to an Obelisk is a two-way Astral Gate, which only becomes active after that Obelisk has been visited.
  \item An \textbf{Astral Gate} cannot be entered from a different Map Tile. When a Hero enters an Astral Gate, they are teleported to a different, randomly selected activated Astral Gate on the Map, arriving on a Field adjacent to it.
  \item Instead of teleporting randomly, a player may reduce 1 Unit's \svg{health_points} to 0 to choose the destination Astral Gate.
\end{itemize}

\vspace*{\fill}
\begin{center}
  \includegraphics[width=1.16\linewidth]{\maps/astral_run.png}
  \captionof{figure}{\textbf{SCENARIO MAP LAYOUT}}
\end{center}

\columnbreak

\begin{itemize}
  \item Gain 1 \svg{experience} after a Combat against an enemy Main Hero.
  \item A defeated Main Hero may Empower one Defense Statistic Card from their M\&M Deck.
  \item \textbf{Level VII Settlement:} Increase all incomes by 1 space.
  \item \textbf{Random Town:} You may recruit one Faction Unit you defeated for its recruitment cost.
  \item \textbf{Dragon Utopia:} You may recruit one dragon you defeated for half of its recruitment cost.

>   \item \textbf{11 Round game:} Remove P × (IV-V) Map Tiles and replace the (VI-VII) Tile with a (IV-V) Map Tile without Obelisk.

Reviewer: ```suggestion
  \item \textbf{11-Round game:} Remove P × Near (IV-V) Map Tiles and replace the Center (VI-VII) Tile with a Near (IV-V) Map Tile without an Obelisk.
```

  \item Instead of teleporting randomly, a player may reduce 1 Unit's \svg{health_points} to 0 to choose the destination Astral Gate.
\end{itemize}
  
  \begin{center}
  \vspace*{\fill}
  \includegraphics[width=1.1\linewidth]{\maps/astral_run.png}
  \captionof{figure}{\textbf{SCENARIO MAP LAYOUT}}
  \vspace*{\fill}
  \end{center}
  
\columnbreak

\begin{itemize}
  \item Gain 1 \svg{experience} after a Combat against an enemy Main Hero.
  \item A defeated Main Hero may Empower one Defense Statistic Card from their M\&M Deck.
  \item \textbf{Level VII Settlement:} Increase all incomes by 1 space.
  \item \textbf{Random Town:} You may recruit one Faction Unit you defeated for its recruitment cost.
  \item \textbf{Dragon Utopia:} You may recruit one dragon you defeated for half of its recruitment cost.
  \item \textbf{11 Round game:} Remove P × (IV-V) Map Tiles and replace the (VI-VII) Tile with a (IV-V) Map Tile without Obelisk.
\end{itemize}

\begin{center}

>   \vspace*{\fill}

Reviewer: ```suggestion
```

  \vspace*{\fill}
  \includegraphics[width=1.1\linewidth]{\maps/astral_run_short.png}
  \captionof{figure}{\textbf{SHORT GAME MAP LAYOUT}}
  \vspace*{\fill}
\end{center}

> \end{multicols}\\

Reviewer: ```suggestion
\end{multicols}
```
